dashboard/src/components/MessageDetails/MessageDetails.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import * as Modal from "react-modal";
+
+import "./MessageDetails.css";
+
+interface IMessageDetailsProps {
+  modalIsOpen: boolean;
+  closeModal: () => Promise<any>;
+  message: string;
+}
+
+const MessageDetails: React.SFC<IMessageDetailsProps> = props => {
+  return (
+    <div className="MessageDetails">
+      <Modal
+        isOpen={props.modalIsOpen}
+        onRequestClose={props.closeModal}
+        contentLabel="Modal"
+        className="Terminal"
+        style={{
+          overlay: {
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+          },
+          content: {
+            maxWidth: "80%",
+          },
+        }}
+      >
+        <div className="Terminal__Top type-small">
+          <div className="Terminal__Top__Buttons">
+            <a>
+              <span
+                className="Terminal__Top__Button Terminal__Top__Button--red"
+                onClick={props.closeModal}
+              />
+            </a>
+          </div>
+          <div className="Terminal__Top__Title">Message</div>
```

Comment:
[Not a blocker] Could you change this title to something more specific? I think `Message` is to general.

Reply:
+1 I would make it a prop so that it can be set by the caller